templates/content/taxonomy-term--localgov-blog-author.html.twig

```diff
@@ -25,13 +25,12 @@
 #}
 {% if not localgov_base_remove_css %}
   {{ attach_library('localgov_base/term') }}
-  {#{{ attach_library('localgov_base/blog') }}#}
 {% endif %}
 
 <div{{ attributes }}>
```

Comment:
We'll need an 'lgd-term' class in here: `<div{{ attributes.addClass('lgd-term') }}>`